### ISSUE TITLE
Mynode FREEZES every few days ???

### ISSUE THREAD

**Issue by salenjak** [mynodebtc/mynode#293]
As I described in the title, for some reason every few days (I believe that one week without freezing is maximum uptime for my device) Mynode is freezing for some reason.

RPI is working, the internet is ON, but the device is unreachable via mynode.local, 192.168.1.x, and of corse wallets, in my case Electrum and Samourai. This also means that just like this morning when I need to use Bitcoin, I can't, because after I manually restarted device on power switch, I need to wait to synchronise again. Now it is behind some 400 blocks.

Anyway, I would appreciate if someone can point me how to debug this to see is it a hardware or software issue. I have downloaded mynode_logs.tar, but not sure where to look .

Thanks in advance

**Comment by tehelsper** [comment]
That typically means something is running improperly and trying to take up too much RAM. What type of device do you have?

4GB is now recommended if you want to enable and use Dojo, but it can all be run on 2GB (for now) if everything is running properly.

**Comment by salenjak** [comment]
I have 2GB and 1TB HDD. Yes, I noticed in the last few weeks while using Dojo & Whirpool that if I disable Electrum server everything starts to work smoothly because RAM usage goes from 80-90% to 40-50%.
I have purchased the device at https://www.cryptocloaks.com/product/build-a-node/. I am not sure but believe that it is your website so maybe it will be best to warn people there that 2GB RAM option is not recommended.

Anyway, thanks for helping.
